velox/exec/tests/utils/VectorTestBase.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/vector/FlatVector.h"
+#include "velox/vector/tests/VectorMaker.h"
+
+namespace facebook::velox::exec::test {
+
+class VectorTestBase {
```

Comment:
Would it make sense to move this class to velox/vector/tests/ and include it in velox_vector_test_lib library? It would be nice to introduce this class and refactor OperatorTestBase and FunctionTestBase in a separate PR.